coursera/downloaders.py

```diff
@@ -61,6 +61,13 @@ def download(self, url, filename, resume=False):
 
         logging.debug('filename after get parameter removal: <%s>.', filename)
 
+        # FIXME (rbrito): This is pretty arbitrary and quite possibly wrong,
+        # as we don't even take extensions into account.
+        if len(filename) > 255:
+            filename = filename[:252] + '...'
```

Comment:
I think I did the same in `get_lecture_filename`:
https://github.com/coursera-dl/coursera-dl/commit/9d29d1d19e27e2b13238499039326e515de32741